### DISCUSSION TITLE
Customize font family for fluent ui v9

### DISCUSSION THREAD

**Discussion by tienlx97** [microsoft/fluentui#28202] [Q&A]
How to apply font family for all component in fluent ui

**Comment by blendsdk** [comment]
I am not an expert here, but I think you need to create a custom theme, or derive from an existing one.

[Take a look at line 18 here](https://github.com/microsoft/fluentui/blob/e968c73521601209372b661dc925528da6bfa7e3/packages/tokens/src/utils/createDarkTheme.ts#LL18C21-L18C21)

and 

[line 38 here](https://github.com/microsoft/fluentui/blob/e968c73521601209372b661dc925528da6bfa7e3/packages/tokens/src/global/fonts.ts#L38)

I hope it helps.
